build.zig

```diff
@@ -30,13 +26,20 @@ pub fn build(b: *std.Build) void {
         .target = target,
         .optimize = optimize,
     });
-    
 
-    exe.linkLibC();
+    raylib.addTo(b, exe, target, optimize);
+
+    if (target.getOsTag() == .linux) {
+        // const source_files = [_][]const u8{
+        //     "vendor/sod/sod.c",
+        // };
+        // const flags = [_][]const u8{"-std=c99"};
 
-    exe.addIncludePath(LazyPath.relative("sod"));
+        // exe.addIncludePath(LazyPath.relative("vendor/sod"));
+        // exe.addCSourceFiles(&source_files, &flags);
 
-    exe.addCSourceFiles(&source_files, &flags);
```

Comment:
Temporarily commented out because `addCSourceFiles()` behaves differently on `nightly` and I couldn't find a quick solution.